### PR TITLE
V0.55.0 & Repro

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

**Comment by fepitre** [comment]
PipelineRetry

**Comment by marmarek** [comment]
Is this ready for review&merge already? I see green CI.

**Comment by fepitre** [comment]
PipelineRetry

**Comment by fepitre** [comment]
> Is this ready for review&merge already? I see green CI.

Let me retry first to be sure it's not a random reprotest parameter who fixes previous failing libdnf tests in reprotest.

**Comment by fepitre** [comment]
@marmarek I've ran two times the reprotest on the CI with success. Locally the same with --auto-build. I think we are finally good!

File: debian-pkg/debian/patches/0012-data-workaround-for-hardcoded-absolute-path-data-in-.patch [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 508d30bd..35f911e1 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -134,7 +134,7 @@ add_definitions(-DG_LOG_DOMAIN=\\"libdnf\\")
+ add_definitions(-D_FILE_OFFSET_BITS=64)
+ 
+ # tests
+-add_definitions(-DTESTDATADIR=\\"${CMAKE_SOURCE_DIR}/data/tests\\")
++add_definitions(-DTESTDATADIR=\\"/tmp/data/tests\\")
```
Comment (marmarek) [inline debian-pkg/debian/patches/0012-data-workaround-for-hardcoded-absolute-path-data-in-.patch:23]: Do the tests expect some initial data in this directory?

Comment (marmarek) [reply debian-pkg/debian/patches/0012-data-workaround-for-hardcoded-absolute-path-data-in-.patch:23]: nvm, I see the line added below.

**Review by marmarek — APPROVED** [review]

**Pull request merged**